debian/control

```diff
@@ -42,7 +42,6 @@ Description: Header files for UCX library
 Package: libucx0
 Section: libs
 Depends: ${shlibs:Depends}, ${misc:Depends}
-Breaks: libopenmpi3 ( << 4.1.0-3)
```

Comment:
I'd rather see this commented out with a comment that **we** did that, not upstream, and why.  Just that in the future, we know.